src/utilities/shouldSetAlarm/shouldSetAlarm.test.js

```diff
@@ -0,0 +1,32 @@
+import { shouldSetAlarm } from './shouldSetAlarm';
+
+describe('The shouldSetAlarm function', () => {
+  describe('when isEmployed parameter is set to true', () => {
+    describe('when isOnVacation parameter is set to false', () => {
+      it('should return true', () => {
+        const result = shouldSetAlarm(true, false);
```

Comment:
It is super difficult to know which boolean is `isEmployed`, and which one is `isOnVacation`. It is very easy to make a mistake here. This is not how this function was supposed to work like.